src/ScssAssetConverter.php

```diff
@@ -101,8 +117,21 @@ private function shouldConvert(string $inFile, string $outFile): bool
         }
     }
 
-    private function isOlder(string $fileA, string $fileB): bool
+    protected function isOlder(string $fileA, string $fileB): bool
     {
         return $this->storage->getMtime($fileA) < $this->storage->getMtime($fileB);
     }
+    
+    protected function getScssPhpFormatterClass($formatter): string
+    {
+        if (!isset($this->scssPhpFormatterMap[$formatter]))
+        {
+            //use default formatter if not found
+            $formatter = $this->defaultFormatter;
+
+            //if (YII_DEBUG) Yii::error("Formatter $formatter not found.", __METHOD__);
```

Comment:
Why is this commented out? I think it makes sense to enable it.

Reply:
seem no more defaultFormtter I have remove the method 'getScssPhpFormatterClass'